satellite/console/consoleweb/server.go

```diff
@@ -87,11 +89,16 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 	}
 
 	mux := http.NewServeMux()
-	fs := http.FileServer(http.Dir(server.config.StaticDir))
+	zipfile, err := zipfs.New(server.config.StaticArchive)
+	if err != nil {
+		logger.Fatal(err.Error())
```

Comment:
this should return an error. This means you'll want to change `NewServer` to `OpenServer` and make it return `*Server, error`